### ISSUE TITLE
[$10][MSFT-47] [Login] "Forgot your password?" hyperlink foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2768]
**Steps to Reproduce**
1. Go to Go to https://accounts.topcoder.com/member
2. Check the "Forgot your password?" hyperlink

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #2C2C2C. New contrast ratio: 4.53
- Passes AA, if you change text color to #0079CE. New contrast ratio: 4.54

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 3.08 (foreground color: #0096ff, background color: #ffffff, font size: 9.0pt, font weight: normal)

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F26

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2768 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2768 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2768 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2768 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096334 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096334 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/229

**Comment by tosha5252** [comment]
Per DaraK's comment on another ticket, the color should be changed to #006DEA , instead of #0079ce...
can you please change and re-submit?  Thanks

Reference ticket:
https://github.com/topcoder-platform/community-app/issues/2888

Env:

WIndows 10
Browser: Chrome

**Comment by PrakashDurlabhji** [comment]
@veshu @tosha5252 new PR raised https://github.com/appirio-tech/accounts-app/pull/260

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097510 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097510 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
Color has been changed to #0096ff

Per DaraK's comment on another ticket, the color should be changed to #006DEA

Reference ticket:
#2888

Env:
WIndows 10
Browser: Chrome

**Comment by PrakashDurlabhji** [comment]
@veshu prize for this ticket was not doubled while for other issues it was doubled. can you please do it if possible?

**Comment by PrakashDurlabhji** [comment]
@tosha5252 @veshu @codeMinter new PR according to above comment. https://github.com/appirio-tech/accounts-app/pull/273

**Comment by tosha5252** [comment]
verified changed

Env:
WIndows 10
Browser: Chrome

**Comment by ajefts** [comment]
Verified in prod:

![image](https://user-images.githubusercontent.com/3812270/62153959-b7065580-b2d3-11e9-9ac6-0e54642c9ebb.png)

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097510<br/><br/>```This is an automated message for crazyk via Topcoder X```
